test/Microsoft.Health.Fhir.Tests.E2E/Crucible/CrucibleDataSource.cs

```diff
@@ -7,12 +7,13 @@
 using System.Linq;
 using System.Net.Http;
 using System.Threading.Tasks;
-using Hl7.Fhir.Rest;
+using Microsoft.Health.Fhir.Tests.Common.FixtureParameters;
 using Microsoft.Health.Fhir.Tests.E2E.Crucible.Client;
-using FhirClient = Microsoft.Health.Fhir.Tests.E2E.Common.FhirClient;
 
 namespace Microsoft.Health.Fhir.Tests.E2E.Crucible
 {
+    extern alias E2EStu3;
```

Comment:
Does crucible tests need to be decorated with the FhirVersion.Stu3 so it only runs for STU3?